.github/workflows/build.yml

```diff
@@ -379,19 +382,13 @@ jobs:
           name: ${{ env.PKG_NAME }}_${{ needs.get-product-version.outputs.product-version }}_linux_${{ matrix.arch }}.zip
 
       # build docker image
-      - name: Docker build
-        working-directory: ./.release/docker
-        run: |
-          docker version
-          unzip -j ${GITHUB_WORKSPACE}/${{ env.PKG_NAME }}_${{ needs.get-product-version.outputs.product-version }}_linux_${{ matrix.arch }}.zip
-          # arm32 bit docker images are set with --platform=linux/arm/v6
-          if [ "${{ matrix.arch }}" = "arm" ]; then
-            make docker ARCH=${{ matrix.arch }} GOARM_DOCKER="/v6"
-          else
-            make docker ARCH=${{ matrix.arch }}
-          fi
-      # upload image tarball
-      - uses: actions/upload-artifact@v2
+      - name: Docker build (Action)
+        uses: hashicorp/actions-docker-build@v1
         with:
-          name: ${{ env.PKG_NAME }}_${{ needs.get-product-version.outputs.product-version }}_docker_linux_${{ matrix.arch }}.tar
-          path: .release/docker/${{ env.PKG_NAME }}_${{ needs.get-product-version.outputs.product-version }}_docker_linux_${{ matrix.arch }}.tar
+          version: ${{env.version}}
+          target: default
+          dockerfile: .release/docker/Dockerfile
+          arch: ${{matrix.arch}}
+          tags: |
+            docker.io/hashicorp/${{env.repo}}:${{env.version}}
+            ecr.public.aws/hashicorp/${{env.repo}}:${{env.version}}
```

Comment:
I wonder if we should add handling for the `latest` tag here as well for the short term? (This isn't documented anywhere yet, and I think @sarahethompson was working on handling latest automatically so we wouldn't need to do this, but I'm not sure if that's ready yet. Can we hold out on merging this until she's back in tomorrow?)

Reply:
@samsalisbury  I'm definitely OK holding this off until @sarahethompson gets back.

Reply:
I think it makes sense to merge asap since the latest tag changes will be made at publish time and this brings CTS up to date with what we have in the Consul and Vault repos (using the docker action to build the docker images).